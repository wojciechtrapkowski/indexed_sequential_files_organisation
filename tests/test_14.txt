insert 1 100
print
update 1 1000
search 1
print