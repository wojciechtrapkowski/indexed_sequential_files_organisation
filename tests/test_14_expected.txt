
================================================
Index area
================================================
Page 0 number of entries: 1
	Entry 0
		start_key: 1
		page_index: 0
================================================
Main area
================================================
Guardian overflow page index: null

Page 0 number of entries: 1
	Entry 0
		key: 1
		value: 100
		overflow_entry_index: null
================================================
Overflow area
================================================
Page 0 number of entries: 0
1000
================================================
Index area
================================================
Page 0 number of entries: 1
	Entry 0
		start_key: 1
		page_index: 0
================================================
Main area
================================================
Guardian overflow page index: null

Page 0 number of entries: 1
	Entry 0
		key: 1
		value: 1000
		overflow_entry_index: null
================================================
Overflow area
================================================
Page 0 number of entries: 0
